### ISSUE TITLE
Scoring 'content-free' applications and frameworks

### ISSUE THREAD

**Issue by johnfoliot** [w3c/silver#494]
The [current draft states](https://www.w3.org/TR/wcag-3.0/#conformance-levels-0):

> To do this, the conformance model **prioritizes content needed to complete tasks** while still testing the entire view for accessibility errors

How will this work for content-free applications such as WordPress, Drupal, SharePoint, Adobe Experience Manager (and other CMS'es), or applications like Blackboard (LMS) or PeopleSoft and similar web-based tools (applications that may include human resource management systems (HRMS), customer relationship management (CRM), financials and supply chain management (FSCM) and enterprise performance management (EPM) tools)? These 'out-of-the-box' web and cloud-based applications will also need to be able to publish accessibility conformance reports (VPATs being one such form) that is written explicitly devoid of any "content" - as "content" will be added after the fact (after purchase) by the end 'owner'.

Additionally, how will organizations be able to evaluate Design Systems and/or Component Libraries that are intentionally created "content-free" - the primitive building blocks that will result in 'pages' and 'task-flows'? What about 3rd-party "[Accessible Themes](https://wordpress.com/themes/filter/accessibility-ready)" for these types of tools as well?

Yes, these tools will still be able to evaluate "the entire view", but because the focus of the scoring model is on content, these tools will not be able to "score" as highly as end products **with** content, as the scoring model "...**prioritizes content needed to complete tasks**..." 

It would be extremely troubling that a "Completely accessible" out-of-the-box CMS would score lower than the same CMS with additional inaccessible content added to the application. 

The current draft does not appear to address this concern.

**Comment by jspellman** [comment]
I don't think there is anything in the current draft that precludes this issue.  The guidelines can apply or not, if they do not apply they are not included in the average.  The devil is in the details, of course, but I think we need to develop more guidelines or some detailed use cases before we can thoroughly determine whether or not the current FPWD addresses them.

**Comment by johnfoliot** [comment]
If, as the draft states "...the conformance model **prioritizes content** needed to complete tasks" and I have content free applications, then I cannot 'prioritize' content as by design my app is content free.  The current language is asking me to compare my apple to your orange: content free apps will score lower than content-rich sites because of the content, which is 'prioritized'.

**Comment by jspellman** [comment]
I think that is in the details.  Potentially components could score higher because the #2 critical error generally (but not always) wouldn't apply.  We probably need to look at the language that says we "prioritize content needed to complete tasks" because it implies more than we intended writing it.

**Comment by slauriat** [comment]
Draft response:

Definitely something that we'll continue working through, and the continuation of establishing how basic conformance and scoring will work can help set some of the foundations that we'll need before we can really answer this more completely.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
